taskfiles/deps.yaml

```
@@ -3,18 +3,56 @@ version: "3"
 vars:
   G_CATCH2_LIB_NAME: "Catch2"
   G_CATCH2_WORK_DIR: "{{.G_DEPS_DIR}}/{{.G_CATCH2_LIB_NAME}}"
+  G_OUTCOME_LIB_NAME: "outcome"
+  G_OUTCOME_WORK_DIR: "{{.G_DEPS_DIR}}/{{.G_OUTCOME_LIB_NAME}}"
+  G_QUICKCPPLIB_LIB_NAME: "quickcpplib"
+  G_QUICKCPPLIB_WORK_DIR: "{{.G_DEPS_DIR}}/{{.G_QUICKCPPLIB_LIB_NAME}}"
 
 tasks:
   install-all:
     desc: "Install all dependencies required by ystdlib-cpp."
+    run: "once"
+    cmds:
+      - task: "install-all-init"
+      - task: "install-all-run"
+      - task: "install-all-finish"
+
+  install-all-init:
+    internal: true
+    cmds:
+      - "rm -rf {{.G_DEPS_CMAKE_SETTINGS_DIR}}"
+      - "mkdir -p {{.G_DEPS_CMAKE_SETTINGS_DIR}}"
+
+  install-all-run:
+    internal: true
     deps:
       - "install-Catch2"
+      - "install-outcome"
+
+  install-all-finish:
+    internal: true
+    cmds:
+      - >-
+        for file in {{.G_DEPS_CMAKE_SETTINGS_DIR}}/*.cmake; do
+          if [ "$file" != "{{.G_DEPS_CMAKE_SETTINGS_FILE}}" ]; then
+            echo "include($file)" >> "{{.G_DEPS_CMAKE_SETTINGS_FILE}}";
```

Lets quote the file name.

```suggestion
            echo "include(\"$file\")" >> "{{.G_DEPS_CMAKE_SETTINGS_FILE}}";
```